test/modules/aggregation-layers/contour-layer/contour-layer.spec.js

```diff
@@ -60,7 +60,7 @@ test('ContourLayer#renderSubLayer', t => {
   t.end();
 });
 
-test('ContourLayer#updates', t => {
+test.skip('ContourLayer#updates', t => {
```

Comment:
comment about why this needed to be skipped?

Reply:
See PR summary